### ISSUE TITLE
Network Error removes "Next" Button on student registration

### ISSUE THREAD

**Issue by TheLightUpMario** [RiseCareerAcademy/Rise#210]
### Do you want to request a *feature* or report a *bug*?
Bug
### How to Reproduce
Attempt to register a student account, filling in all available fields. Hit "next." After a network error bug notification appears, use the back button to navigate back to the main page and click on register a student account again. 
### What is the current behavior? (Include which device (Android/iOS).)**
On android. Every time the next button is clicked on the registration page, a network error occurs. When the user goes to the main screen and tries to register again, the next button is gone entirely and instead a buffering animation plays. 
### What is the expected behavior?
The next button succeeds, and failing that, restarting the student registration process should have the next button to attempt again.

**Comment by kmui2** [comment]
What time exactly did you have this issue? I just restarted the server which may have caused this network error.

**Comment by TheLightUpMario** [comment]
I just attempted again and got the error.

**Comment by TheLightUpMario** [comment]
Yes, I used expo, logged in with the info, and scanned the linked qr code.
